test/build/55_cache.bats

```diff
@@ -1112,7 +1112,7 @@ EOF
     # happening on a tmpfs.
     ch-image build -t tmpimg - <<'EOF'
 FROM alpine:3.17
-RUN for i in $(seq 0 1024); do \
+RUN for i in $(seq 0 1536); do \
```

Comment:
It isn't clear why this changed to 1536 at a glance, perhaps a comment?

Reply:
Yeah, it's a race condition we keep fiddling with. See new issue #1587.